cohortfinder/test/histoqc_outdir/Untitled.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
should this file be in the repo, if yes, can it be renamed accordingly?

Reply:
@FanFAN0703  can you comment on this?